src/main/scala/com/horizon/exchangeapi/BusinessRoutes.scala

```diff
@@ -379,6 +381,17 @@ trait BusinessRoutes extends JacksonSupport with AuthenticationSupport {
       ]
     }
   ],
+  "secretBinding": [
+     {
+       "serviceOrgid": "string",
+        "serviceUrl": "string",
+        "serviceArch": "amd64",
+        "serviceVersionRange": "x.y.z",
+         "secrets": [
+         { "vaultSecret": "string" }
```

Comment:
is vaultSecret a variable or is it an explicit key/field name? The design says that it is a variable, essentially just a string.

Reply:
@dabooz It is just a variable.